### PR TITLE
Perspective camera

### PR DESCRIPTION
**Pull request by ykulbak**
Requires testing with Solibri and Catenda

### CODE REVIEW

**Comment by cschladetsch** [comment]
There are inconsistent line endings for `Test Cases/v3.0/Visualization/Perspective camera/README.md`. Some bullet points end in a full stop, others do not. Trivial, yes.

File: Test Cases/v3.0/Visualization/Perspective camera/unzipped/01777b21-ba39-4c2b-ad1d-a9320f81214a/formatted [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/Perspective camera/unzipped/01777b21-ba39-4c2b-ad1d-a9320f81214a/formatted:1]: Looks like an unnecessary file.

File: Test Cases/v3.0/Visualization/Perspective camera/unzipped/bcf.version [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Version VersionId="3.0">
+  <DetailedVersion>3.0</DetailedVersion>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/Perspective camera/unzipped/bcf.version:3]: Detailef version shouldn't be here.

**Comment by leosobral** [comment]
Tested Ok in Bimsync, the red part of the snapshot image is not showing in Bimsync.

**Review by pasi-paasiala — CHANGES_REQUESTED** [review]
-1

**Comment by pasi-paasiala** [comment]
Was able to read the BCF, but there are still some mistakes.

**Comment by ykulbak** [comment]
> There are inconsistent line endings for `Test Cases/v3.0/Visualization/Perspective camera/README.md`. Some bullet points end in a full stop, others do not. Trivial, yes.
> 
> Also, do we want spaces in pathnames and should they have a mixed case?
> 
> For coders and cross-platform ease, the pathname `TestCases/v3.0/Visualization/PerspectiveCamera/Readme.md` would be preferred.
> 
> Unsure what the convention for README.md vs. Readme.md is.

@cschladetsch thanks for the comments. The file and directory naming conventions are as discussed an agreed with the group. I believe I fixed the inconsistencies in the README. 

@pasi-paasiala @jasollien review comments applied, please have a look

**Comment by CBenghi** [comment]
Looks good to me.

**Review by CBenghi — APPROVED** [review]

**Review by pasi-paasiala — APPROVED** [review]

**Pull request merged**